client_linux.go

```diff
@@ -19,6 +20,9 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+// errNotSupported is returned when an operation is not supported
+var errNotSupported = errors.New("not supported")
```

Comment:
Should we export this error so it can be used like `errors.Is(err, wifi.ErrNotSupported)` when calling `wifi.ConnectWPAPSK()`?

Reply:
I will leave this up to you. In `client_others.go` there is `errUnimplemented` which covers a similar error.